src/common/slack.js

```diff
@@ -6,6 +6,21 @@
 const SLACK_HOOK_URL =
   'aHR0cHM6Ly9ob29rcy5zbGFjay5jb20vc2VydmljZXMvVDYyR1g1UlFVL0JCUzUyVDc5OC9UWnZZSGkzcnFyY29aSWYyaFlCVDZVdTA=';
 
+// get IP
+const getIP = async () => {
+  let ip = 'N/A';
+
+  await fetch('https://api.ipify?format=json')
+    .then(resp => resp.json())
```

Comment:
hmm I thought the point of await was to avoid these callbacks. Can we use one or the other? putting the `fetch` inside of a `try...catch` should have the same effect.

Reply:
I thought it was clearer to read than than nested awaits